admin/base/forms.py

```diff
@@ -0,0 +1,5 @@
+from django import forms
+
+
+class GuidForm(forms.Form):
+    guid = forms.CharField(label='Guid', max_length=5, required=True)
```

Comment:
This should probably specify `min_length=5` too, unless we have any guids less than length 5 (Institutions, maybe?).

Also, it might be good to note that `max_length` will need to be updated whenever we bump the guid length up to 6.